README.rst

```diff
@@ -1,5 +1,11 @@
 |OrbDot Logo|
 
+======
+OrbDot
+======
+:Author: Simone Hage
+:Version: 1.0.0
+
```

Comment:
Lukas, you spelled my name wrong. Please spell my name as "Simone R. Hagey" :)

Reply:
done